Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -31,7 +31,7 @@ public enum ManifestParseError: Swift.Error {
 /// using the package manager with alternate toolchains in the future.
 public protocol ManifestResourceProvider {
     /// The path of the swift compiler.
-    var swiftCompiler: AbsolutePath { get }
+    var swiftCompiler: AbsolutePath? { get }
```

Comment:
@SDGGiesbrecht We didn't really need absolute path of the compiler here. As long as the OS shim can find it, we should be fine.